tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,23 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+
+    assert result == True 
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    # refactored
+    assert all(item in [item_d, item_e, item_c] for item in jesse.inventory)
+    assert all(item in [item_a, item_b, item_f] for item in tai.inventory)
```

Comment:
I see you! 🎉 This is a sweet shorthand. You did add an iteration but it would take some time to look up.